### ISSUE TITLE
Swarmpit exit 137

### ISSUE THREAD

**Issue by ducminhnguyen** [swarmpit/swarmpit#323]
**Description**
Swarmpit admin app crashes from time to time, with exit code 137

**Steps to reproduce the issue:**
Deploy swarmpit and wait for crash to happen

**What happens:**
Swarmpit admin_app crash with 137 error

**What should happen:**
Swarmpit admin_app should not crash with 137

**Additional information (e.g. docker version, cluster setup,...):**
```
docker version
Client:
 Version:           18.09.1
 API version:       1.39
 Go version:        go1.10.6
 Git commit:        4c52b90
 Built:             Wed Jan  9 19:35:23 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       6247962
  Built:            Sun Feb 10 03:42:13 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```
Crash log:
```
 docker logs admin_app.1.92o7svt0p8cnlpszdwxyihpku -f
Mar 07, 2019 11:08:16 AM swarmpit.server invoke
INFO: Swarmpit is starting...
Mar 07, 2019 11:08:16 AM swarmpit.database invoke
INFO: Waiting for DB...
Mar 07, 2019 11:08:16 AM swarmpit.database invoke
INFO: ... connected after 0 sec
Mar 07, 2019 11:08:17 AM swarmpit.server invoke
INFO: Swarmpit running on port 8080
Mar 07, 2019 11:08:18 AM swarmpit.setup invoke
INFO: Docker API: 1.39
Mar 07, 2019 11:08:18 AM swarmpit.setup invoke
INFO: Docker ENGINE: 18.09.2
Mar 07, 2019 11:08:18 AM swarmpit.setup invoke
INFO: Docker SOCK: /var/run/docker.sock
```

**Comment by nohaapav** [comment]
There is nothing significant in crash log .. Just make sure you have enough resources. Swarmpit is then running just fine.

**Comment by nohaapav** [comment]
Closing due to inactivity and lack of data to debug. Feel free to open in case problem persist.

**Comment by ztimson** [comment]
I recently switched from 1.8 to the edge version. I opted for a fresh install over migrating & after the fresh install the admin app would randomly start to hang on a white page. I would have to scale the image down and back up to get it to redeploy to get it to start working again. This problem would happen about once a day.

As recommended by @nohaapav I increased the default memory limit from 1024 MiB to 2048 MiB which seems to have fixed the problem for me. The original 1024 limit was set by default during the installation, can we increase this default limit so others don't run into this?
